Makefile

```diff
@@ -47,7 +47,7 @@ ldlibs := -lrt ${LDLIBS}
 cppflags.tbb :=
 ldlibs.tbb := {-L,'-Wl,-rpath='}/usr/local/lib -ltbb
 
-cppflags.moodycamel := -I$(abspath ..)
+cppflags.moodycamel := -I$(abspath ..) -I${abspath ../xenium}
```

Comment:
This change defeats the name of the variable. Not a show stopper, but it is less than ideal.

Reply:
Ah, that was a quick hack that I forgot about...